cypress/integration/console/devices/create-repository.spec.js

```diff
@@ -115,6 +256,41 @@ describe('End device repository create', () => {
     })
 
     describe('Moko', () => {
+      it('displays UI elements in place', () => {
+        cy.findByLabelText('Brand').selectOption('moko')
+        cy.findByLabelText('Model').selectOption('lw001-bg')
+        cy.findByLabelText('Hardware Ver.').selectOption('1.0.1')
+        cy.findByLabelText('Firmware Ver.').selectOption('1.0.1')
+        cy.findByLabelText('Profile (Region)').selectOption('EU_863_870')
```

Comment:
Note that this will make the e2es dependant on a third party.

Reply:
this is known issue

https://github.com/TheThingsNetwork/lorawan-stack/pull/3695

> ideally we shouldnt be using predefined templates that can change at any moment, but given limited time, imo, this is fine. All work on adding proper e2e setup for end devices in general.